server/scripts/create_tables.sql

```diff
@@ -0,0 +1,35 @@
+CREATE TABLE "user" (
+    id       SERIAL PRIMARY KEY,
+    name     VARCHAR,
+    email    VARCHAR UNIQUE,
+    password VARCHAR
+);
+
+CREATE TABLE room (
+    id         SERIAL PRIMARY KEY,
+    name       VARCHAR,
+    creator_id INT REFERENCES "user" (id)
+);
+
+CREATE TYPE content_type as ENUM ('TEXT', 'FILE');
+
+CREATE TABLE message (
+    id            SERIAL PRIMARY KEY,
+    sender_id     INT REFERENCES "user" (id),
+    room_id       INT REFERENCES room (id),
```

Comment:
Сюда лучше добавить `ON DELETE CASCADE` на оба `foreign key` чтобы при удалении отправителя сообщения либо комнаты все связанные сообщения автоматически удалялись базой данных
```suggestion
    sender_id     INT REFERENCES "user" (id) ON DELETE CASCADE,
    room_id       INT REFERENCES room (id) ON DELETE CASCADE,
```